.github/workflows/label-deploy-failed.yml

```
@@ -1,41 +1,44 @@
     runs-on: ubuntu-latest
     if: |
-      github.repository == 'ohcnetwork/care_fe' && github.event.issue.pull_request &&
+      github.repository == 'ohcnetwork/care_fe' &&
+      github.event.issue.pull_request &&
       contains(github.event.comment.body, 'Deploy Preview')
+
     steps:
-      - name: Add 'Deploy-Failed'
-        if: contains(github.event.comment.body, 'failed')
+      - name: Check out the repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)